packages/android_alarm_manager/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+##  0.5.0-null-safety
+
+* Migrate to null safety.
+
```

Comment:
Please revert the android_alarm_manager changes from this PR.

Reply:
@stuartmorgan , Actually I am new to open source, I already had PR of `anroid_alarm_manager` that was being reviewed...I started working on this issue by making a new branch and then pushed the code to upstream. I don't know but it got merged with my last PR. Can you please tell me how to revert this?

Reply:
You could either revert the commits on that branch, or you can check out those files from master and commit the changes. Resources like https://git-scm.com/book/en/v2/ and StackOverflow would be good places to learn more about how to manage branches in `git`.